addon/components/au-modal.hbs

```diff
@@ -18,6 +18,7 @@
         additionalElements=this.additionalElements
         focusTrapOptions=(hash
           initialFocus=this.initialFocus
+          fallbackFocus=this.fallbackFocus
```

Comment:
Weird thing I noticed here is that defining `fallbackFocus` also fixes the ability to focus the `h1` title element with focus-trap if it is empty (as mentioned here: https://github.com/appuniversum/ember-appuniversum/pull/412#issuecomment-1674523425). Not sure why 🤷‍♂️